src/ui/screens/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,192 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+import telemetry 0.30
+
+VPNViewBase {
+    id: root
+    objectName: "settingsTipsAndTricksPage"
+
+    _menuTitle: VPNl18n.TipsAndTricksSettingsEntryLabel
+    _interactive: false
+
+    _viewContentData: VPNTabNavigation {
+        id: tabs
+        Layout.topMargin: -VPNTheme.theme.windowMargin
+        Layout.preferredHeight: root.height - VPNTheme.theme.menuHeight
+        Layout.preferredWidth: root.width
+
+        tabList: ListModel {
+            id: tabButtonList
+            ListElement {
+                tabLabelStringId: "TipsAndTricksTabTitleAll"
+                tabButtonId: "tabAll"
+            }
+            ListElement {
+                tabLabelStringId:"TipsAndTricksTabTitleTutorials"
+                tabButtonId: "tabTutorials"
+            }
+            ListElement {
+                tabLabelStringId:"TipsAndTricksTabTitleTips"
+                tabButtonId: "tabTips"
+            }
+        }
+
+        ObjectModel {
+            id: tipsSections
+
+            Item { 
+                property string title: VPNl18n.TipsAndTricksQuickTipsTitle
+                property string description: VPNl18n.TipsAndTricksQuickTipsDescription
+                property var filter: (addon) => !addon.advanced && !addon.highlighted
+            }
+
+            Item { 
+                property string title: VPNl18n.TipsAndTricksAdvancedTipsTitle
+                property string description: VPNl18n.TipsAndTricksAdvancedTipsDescription
+                property var filter: (addon) => addon.advanced && !addon.highlighted
+            }
+        }
+
+        stackContent: [
+            // All
+            VPNFlickable {
+                flickContentHeight: layoutAll.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+                interactive: flickContentHeight > height
+
+                ColumnLayout {
+                    id: layoutAll
```

Comment:
Without these `id`'s on each of the ColumLayouts the whole view breaks dows. I don't understand why, these are not used anywhere (by me).

Reply:
`layoutAll` is used 4 lines above `flickContentHeight: layoutAll.implicitHeight + (VPNTheme.theme.vSpacing * 2)`

Reply:
...No comment on myself.